com.unity.multiplayer.mlapi/Tests/Runtime/RpcQueueTests.cs

```diff
@@ -24,9 +24,6 @@ public class RpcQueueTests : IDisposable
         [UnityTest]
         public IEnumerator UpdateStagesInvocation()
         {
- // Disabling this test on 2019.4 due to ILPP issues on Yamato CI/CD runs (UNITY_2020_2_OR_NEWER)
- // Adding the ability to test if we are running in editor local (UNIT_MANUAL_TESTING)
-#if UNITY_2020_2_OR_NEWER || UNITY_MANUAL_TESTING
```

Comment:
changes in this file are coming from: #645 — fix: Resolve an issue where 2019.4 would not properly run ILPP with a clean import